<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_1">
         <property name="title">
          <string>Input</string>
         </property>
         <widget class="QScrollArea" name="scrollArea_3">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>135</y>
            <width>441</width>
            <height>431</height>
           </rect>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>433</width>
             <height>423</height>
            </rect>
           </property>
           <widget class="QPlainTextEdit" name="inpUserPrompt">
            <property name="geometry">
             <rect>
              <x>-5</x>
              <y>-5</y>
              <width>446</width>
              <height>436</height>
             </rect>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QScrollArea" name="scrollArea">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>50</y>
            <width>436</width>
            <height>56</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>428</width>
             <height>48</height>
            </rect>
           </property>
           <widget class="QPlainTextEdit" name="inpSystemPrompt">
            <property name="geometry">
             <rect>
              <x>-5</x>
              <y>-5</y>
              <width>437</width>
              <height>57</height>
             </rect>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>57</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>System: </string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>115</y>
            <width>57</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>User:</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Output</string>
         </property>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>15</y>
            <width>436</width>
            <height>551</height>
           </rect>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>428</width>
             <height>543</height>
            </rect>
           </property>
           <widget class="QPlainTextEdit" name="inpOutput">
            <property name="geometry">
             <rect>
              <x>-5</x>
              <y>-5</y>
              <width>441</width>
              <height>556</height>
             </rect>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>901</width>
          <height>28</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Model name: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="drpModelName">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>20</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>gpt-4-32k</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>gpt-4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>gpt-3.5-turbo-16k</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>gpt-3.5-turbo</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>text-davinci-003</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>text-curie-001</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>text-babbage-001</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>text-ada-001</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>API Key: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inpApiKey">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnSubmit">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Submit</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
